scrapers/AdultEmpire.yml

```diff
@@ -12,8 +12,40 @@ sceneByURL:
       - adultdvdempire.com
       - adultempire.com
     scraper: sceneScraper
+sceneByName:
+  action: scrapeXPath
+  queryURL: https://www.adultdvdempire.com/allsearch/search?q={}
+  scraper: sceneSearch
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+
 
 xPathScrapers:
+  sceneSearch:
+    scene:
+      Title:
+        selector: //div[@id="content"]//div[@class="item-title"]/a/@href
+        postProcess:
+          - replace:
+            - regex: .+?(\d+)/(.+)
+              with: $2 ($1)
+            - regex: "-"
+              with: " "
+            - regex: "/"
+              with: ""
+            - regex: ".html"
+              with: ""
```

Comment:
i have been playing with something like this
```yaml
      Title:
        selector: //div[@class="item-title"]/a/@*[local-name()="href" or local-name()="title"]
        concat: "|"
        postProcess:
          - replace:
            - regex: '/(\d+)/[^|]+\|([^|]+)'
              with: "$2 ($1)"
        split: "|"
```
It seems to work ok and the titles look better imo